qa/common/gen_qa_model_repository

```diff
@@ -286,7 +286,7 @@ python3 $VOLUME_SRCDIR/gen_qa_dyna_sequence_implicit_models.py --onnx --onnx_ops
 chmod -R 777 $VOLUME_DYNASEQIMPLICITDESTDIR
 python3 $VOLUME_SRCDIR/gen_qa_ragged_models.py --onnx --onnx_opset=$ONNX_OPSET --models_dir=$VOLUME_RAGGEDDESTDIR
 chmod -R 777 $VOLUME_RAGGEDDESTDIR
-python3 $VOLUME_SRCDIR/gen_qa_ort_scalar_models.py --onnx_opset=$ONNX_OPSET --models_dir=$SCALARMODELSDESTDIR
+python3 $VOLUME_SRCDIR/gen_qa_ort_scalar_models.py --onnx_opset=$ONNX_OPSET --models_dir=$VOLUME_SCALARMODELSDESTDIR
 chmod -R 777 $VOLUME_RAGGEDDESTDIR
```

Comment:
Should the next `chmod` line 290 also be updated?
```
chmod -R 777 $VOLUME_SCALARMODELSDESTDIR
```

Reply:
For consistency probably you are right.